MidiPlayer/src/main/java/org/primesoft/midiplayer/configuration/ConfigProvider.java

```diff
@@ -186,12 +188,16 @@ public static boolean load(JavaPlugin plugin) {
     }
 
     private static SoundCategory parseSoundCategory(String categoryName) {
-        for (SoundCategory c : SoundCategory.values()) {
-            if (categoryName.equalsIgnoreCase(c.toString())) {
-                return c;
+        if (categoryName != null) {
+            categoryName = categoryName.trim().toUpperCase();
+            for (SoundCategory c : SoundCategory.values()) {
+                if (categoryName.equals(c.name())) {
```

Comment:
Trim is ok, but still using ignore case is better in case of API changes.

Reply:
Removed the ToUpperCase and changed equals to equalsIgnoreCase. Who knows when will Mojang decide to change everything to lowercase...